### ISSUE TITLE
Statistics: Incorrect normalization of graphs

### ISSUE THREAD

**Issue by flagadum** [AdguardTeam/AdguardForAndroid#4564]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

4.0.732

### Environment

```markdown
- OS: Android 8
- Device: CAT S41
- Firmware: Mediatek
```

### Root access

- [ ] Yes, I have it.

### What filters do you have enabled?

Irrelevant

### What Stealth Mode options do you have enabled?

Irrelevant

### Issue Details

Steps to reproduce:
1. Go to the statistics
2. Select either 7 days, 30 days, All time duration
3. For any Requests or Data usage value, click it to display the timeline graph

### Expected Behavior

For a fairly constant usage along time, we would expect the graph do display a roughly similar average height along the whole duration (regardless to the local bumps to be expected here and there).

### Actual Behavior

For almost any of the data displayed, the recent (right) side of the curve (where the sampling rate is fine) always appears  significantly lower than the older (left) side of the curve (where the sampling rate is coarser).

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

### Additional Information

It looks like the curve isn't properly normalized in regard to the variable time duration of the samples. What should actually be plotted as height of a curve point is not the **total data** occurring during the duration of the sample, but the **time average** (hence, the total data divided by the duration of the sample).

Doing so will restore what is intuitively expected from these curves: A constant data rate will lead to a fairly flat curve regardless of the sampling rate being fine or coarse, and we could visually assume that the total data handled during a time interval is roughly represented by the area below the curve within that interval.

**Comment by maxikuzmin** [comment]
@flagadum thanks for your observation. We'll check it out.

**Comment by maxikuzmin** [comment]
@flagadum thanks for the long wait. I have analyzed the stats for 2 months, but the graphs are displaying normally, right side of the curve where the sampling rate is fine don't appears significantly lower than the older left.
Maybe you were blocking requests at the beginning, so the curve went down afterwards. I had the same display when I turned on AdGuard at the beginning of the day, and there were a lot of requests that were blocked, and then there were fewer requests. I think that's the main reason.
This could also depend on the Firewall, for example, because at system startup the Firewall rules are applied and the curve jumps up.
![3](https://github.com/AdguardTeam/AdguardForAndroid/assets/101641415/c124c0f1-0cb0-4e26-ae84-b735717a21ec)
![2](https://github.com/AdguardTeam/AdguardForAndroid/assets/101641415/74818295-6a46-4b0b-8ed5-38a3f7a77395)
![1](https://github.com/AdguardTeam/AdguardForAndroid/assets/101641415/7500fb2c-ce57-4936-bc87-11e40525b593)
![0](https://github.com/AdguardTeam/AdguardForAndroid/assets/101641415/4543df18-a90a-4fec-a40c-0e764c8a18c7)
